"java.io.IOException: Error while instrumenting" error in org.jacoco for Java v21

When executing a Maven "test" target, i.e.:
```bash
mvn test
```

the following errors appear for `java/sql/Timestamp`:

> [INFO] Running org.edmcouncil.rdf_toolkit.writer.SortedJsonLdWriterTest
> java.lang.instrument.IllegalClassFormatException: Error while instrumenting java/sql/Timestamp.
> 	at org.jacoco.agent.rt.internal_3570298.CoverageTransformer.transform(CoverageTransformer.java:94)
> ...
> Caused by: java.lang.IllegalArgumentException: Unsupported class file major version 65

Similar errors also appear for:

- `java/sql/Date`
- `java/sql/Time`

---
Software versions used:

> Apache Maven 3.8.5 (Red Hat 3.8.5-4)
> Maven home: /usr/share/maven
> Java version: 21.0.1, vendor: Red Hat, Inc., runtime: /usr/lib/jvm/java-21-openjdk-21.0.1.0.12-2.0.1.el8.x86_64
> Default locale: pl_PL, platform encoding: UTF-8
> OS name: "linux", version: "5.15.0-202.135.2.el8uek.x86_64", arch: "amd64", family: "unix"

--- a/pom.xml
+++ b/pom.xml
@@ -257,7 +257,7 @@
       <plugin>
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
-        <version>0.8.7</version>
+        <version>0.8.11</version>
         <executions>
           <execution>
             <goals>

